Develop/assets/css/style.css

```diff
@@ -93,6 +108,11 @@ code {
   outline: 2px dashed var(--gin);
   outline-offset: -2px;
   transition: all 0.5s ease-in-out;
+
+/* new */
+  flex: 0 1 95%;
+  margin: 3% auto 1%;
+  padding: 1%;
```

Comment:
Flex-item given: flex-grow = 0, flex-shrink = 1, and flex basis = 95%.
Flex basis may be better set to 100% and spacing be added with additional code but 95% leaves 2.5% width on the sides of the card column.